CMakeLists.txt

```diff
@@ -9,14 +9,32 @@ message(STATUS Preparing\ ${PROJECT_NAME}\ version\ ${PROJECT_VERSION})
 function(num_option name description default)
     option(${name} ${description} ${default})
     if(${name})
+        message(STATUS "[*] ${name}")
         set(${name} 1 PARENT_SCOPE)
     else()
+        message(STATUS "[ ] ${name}")
```

Comment:
Small thing but I love it. Much better than that flag message I've created before!